linear/linear_regression_multiple_features.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+
+class LinearRegressionMultipleFeatures:
+    
+    def __init__(self, n_estimators, learning_rate):
+        self.__n_estimators = n_estimators
+        self.__learning_rate = learning_rate
+        self.m = None
+        self.b = None
+        self.loss_history = []
```

Comment:
I think `m, b and loss_history` can be private?
Also, are you sure that they should be declared in the init?